src/app/components/badges/full-list-of-achievements/full-list-of-achievements.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="full-list">
+    <div class="full-list-header">
+        <h2>FULL LIST OF ACHIEVEMENTS</h2>
+    </div>
+
+    <div class="title">
+        <div class="title1">Achievement</div>
+        <div class="title2">EXP points</div>
+        <div class="title3">Date</div>
+    </div>
+
+    <div class="content" *ngFor="let item of list">
+        <div class="achPic" style="background-image: url({{item.achPic}});"></div>
```

Comment:
1. css classes should not be in camel case.
2. better use full name  like achievement-picture
this goes to all files, not only this